### PR TITLE
Update docs to reflect new package installation workflow.

### PR DESCRIPTION
**Pull request by surfnerd**
- Fix old material name references.
- Update outdated code comments.

### CODE REVIEW

File: docs/Installation.md [hunk 1]
```diff
@@ -39,22 +39,31 @@ The `Project` subdirectory contains many [example environments](Learning-Environ
...
+on the `+` button.
 
-If you intend to copy the `com.unity.ml-agents` folder in to your project, ensure that
-you have the [Barracuda preview package](https://docs.unity3d.com/Packages/com.unity.barracuda@0.3/manual/index.html) installed.
+<p align="center">
+  <img src="images/unity_package_manager_window.png"
+       alt="Linux Build Support"
+       width="500" border="10" />
+</p>
+**NOTE:** In Unity 2018.4 it's on the bottom left and in Unity 2019.3 it's on the top left.
```
Comment (chriselion) [inline docs/Installation.md]: Weird, this shows up as literal `**`'s if you view the file. Maybe the `:` is confusing it?

File: docs/Installation.md [hunk 2]
```diff
@@ -39,22 +39,31 @@ The `Project` subdirectory contains many [example environments](Learning-Environ
...
-       width="710" border="10"
-       height="569" />
+  <img src="images/unity_package_json.png"
+       alt="Linux Build Support"
+       width="500" border="10" />
 </p>
 
+If you are going to follow the examples from our documentation, you can open the `Project`
+folder in Unity and start tinkering immediately.
+
```
Comment (vincentpierre) [inline docs/Installation.md]: There are some project settings that the user should modify in the PlayerSettings no ?

Comment (surfnerd) [reply docs/Installation.md]: Like what?

File: docs/Migrating.md [hunk 3]
```diff
@@ -11,6 +11,7 @@ The versions can be found in
 ## Migrating from 0.13 to latest
 
 ### Important changes
+* The `UnitySDK` folder has been split into a unity package (`com.unity.ml-agents`) and an examples project (`Project`).  Please follow the [Intallation Guide](Installation.md) to get up and running with this new repo structure.
```
Comment (andrewcoh) [inline docs/Migrating.md]: Does "unity" need to be capitalized?

File: docs/Installation.md (hunk 1 above)
Comment (surfnerd) [reply docs/Installation.md]: looks like it needed an extra line between the `</p>` and the `**`

File: docs/Migrating.md (hunk 3 above)
Comment (surfnerd) [reply docs/Migrating.md]: couldn't hurt.

**Pull request merged**